ecscollector/collector.go

```diff
@@ -0,0 +1,270 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I am not familiar with the matter to know the exact impact of copyrighting this to Amazon vs our usual "prometheus contributors" copyright. pinging @SuperQ @RichiH here.

Reply:
Fixed this, it was bad copy/paste.